examples/hello_world/main.rs

```diff
@@ -56,6 +56,7 @@ fn length(db: &impl HelloWorldDatabase, (): ()) -> usize {
 
 // Define the actual database struct. This must contain a salsa
 // runtime but can also contain anything else you need.
+#[salsa::database(HelloWorldDatabase)]
 #[derive(Default)]
```

Comment:
This is an example, so short comment explaining the attribute would be helpful

Reply:
good catch